modules/system/controllers/Settings.php

```diff
@@ -1,5 +1,7 @@
 <?php namespace System\Controllers;
 
+use Config;
+use Request;
```

Comment:
It's a minor quibble I know, but I like having the imports ordered by lengt

Reply:
Changed.